composer.json

```diff
@@ -54,7 +54,8 @@
         "symfony/translation": "4.3.*",
         "symfony/expression-language": "4.3.*",
         "ext-json": "*",
-        "justinrainbow/json-schema": "^5.2"
+        "justinrainbow/json-schema": "^5.2",
+        "ext-pdo": "*"
```

Comment:
what is the purpose of this dependency ?
Cannot find any documentation here : https://github.com/php-kit/ext-pdo

Reply:
seems ok, this is just the check that pdo extension is enabled